specification/analysisservices/resource-manager/Microsoft.AnalysisServices/2017-08-01-beta/analysisservices.json

```diff
@@ -381,6 +381,48 @@
       }
     }
   },
+  "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AnalysisServices/servers/{serverName}/listGatewayStatus": {
+      "post": {
+        "description": "Return the gateway status of the specified Analysis Services server instance.",
+        "tags": [ "Servers" ],
+        "operationId": "Servers_ListGatewayStatus",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "serverName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-z][a-z0-9]*$",
+            "minLength": 3,
+            "maxLength": 63,
+            "description": "The name of the Analysis Services server. It must be at least 3 characters in length, and no more than 63."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": { "$ref": "#/definitions/GatewayListStatusLive" }
+          },
+          "400": {
+            "description": "Gateway not live or not configured.",
+            "schema": { "$ref": "#/definitions/GatewayListStatusError" }
+          },
+          "404": {
```

Comment:
There could be other error codes that this API returns. So its best to capture the error conditions by defining a "default" response and including the error object definition in that.